packages/__docs__/src/ColorBand/index.tsx

```diff
@@ -24,31 +24,22 @@
 
 /** @jsx jsx */
 import { Component } from 'react'
-import PropTypes from 'prop-types'
 
 import { Flex } from '@instructure/ui-flex'
 import { View } from '@instructure/ui-view'
 
 import { withStyle, jsx } from '@instructure/emotion'
 import generateStyle from './styles'
 import generateComponentTheme from './theme'
+import { propTypes, allowedProps } from './props'
+import type { ColorBandProps } from './props'
 
 @withStyle(generateStyle, generateComponentTheme)
-class ColorBand extends Component {
-  static propTypes = {
-    height: PropTypes.string,
-    // eslint-disable-next-line react/require-default-props
-    makeStyles: PropTypes.func,
-    // eslint-disable-next-line react/require-default-props
-    styles: PropTypes.object
-  }
-
+class ColorBand extends Component<ColorBandProps> {
+  static propTypes = propTypes
+  static allowedProps = allowedProps
   componentDidMount() {
-    this.props.makeStyles()
-  }
-
-  componentDidUpdate(prevProps, prevState, snapshot) {
-    this.props.makeStyles()
```

Comment:
Oh soooory, I wasn't clear about this. Please don't delete the whole `componentDidUpdate` method, it is needed for the component, just please delete the unused parameters: 

```js
componentDidUpdate() {
    this.props.makeStyles?.()
}
```